commands/root.go

```diff
@@ -79,7 +79,9 @@ func NewRootCmd(name string, isPlugin bool, dockerCli *command.DockerCli) *cobra
 		cmd.SilenceErrors = true
 		cmd.TraverseChildren = true
 		cmd.DisableFlagsInUseLine = true
-		cli.DisableFlagsInUseLine(cmd)
+		visitAll(cmd, func(c *cobra.Command) {
+			c.DisableFlagsInUseLine = true
```

Comment:
Ideally, we would not be doing this at runtime, but perhaps instead a test that validates all commands have the option set (I  was considering this for the CLI itself as well).